appshell/appshell_extensions.cpp

```diff
@@ -394,10 +399,15 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
             CefWindowInfo wi;
             CefBrowserSettings settings;
 
-#if defined(OS_WIN)
-            wi.SetAsPopup(NULL, "DevTools");
-#endif
-            browser->GetHost()->ShowDevTools(wi, browser->GetHost()->GetClient(), settings, CefPoint());
+            #if defined(OS_WIN)
+                wi.SetAsPopup(NULL, "DevTools");
+            #elif defined(OS_LINUX)
+                handler->ShowDevTools(browser, CefPoint());
+            #endif
+
+            #ifndef OS_LINUX
+                browser->GetHost()->ShowDevTools(wi, browser->GetHost()->GetClient(), settings, CefPoint());
```

Comment:
This statement seems to be executed in case of MAC, so can we have a separate `#elif` case in line 406 only.
Something like `#ifdef OS_MAC`, it would make it more readable.